tests/src/test/scala/cats/tests/RegressionTests.scala

```diff
@@ -85,7 +85,7 @@ class RegressionTests extends CatsSuite {
 
   test("#500: foldMap - traverse consistency") {
     assert(
-      List(1,2,3).traverseU(i => Const(List(i))).getConst == List(1,2,3).foldMap(List(_))
+      List(1,2,3).traverse(i => Const.of[List[Int]](List(i))).getConst == List(1,2,3).foldMap(List(_))
```

Comment:
Here is the minimised version in case anyone is interested. 
Here is the minimized version
```Scala
    def trav[F[_], A, B](f: A => F[B]): F[B] = ???
    case class C[A, B](a: A)
    val c: C[Int, Int] = trav(C(_:Int))   //compiles
    val a: Int  = trav(C(_:Int)).a    //doesn’t compile
```
> [error]  found   : Int => C[Int,Nothing]
[error]  required: Int => C[Int,B]
[error]     val a: Int  = trav(C(_:Int)).a

We might have to live with this as is.